src/predicate/float/close.rs

```diff
@@ -0,0 +1,101 @@
+use float_cmp::ApproxEq;
```

Comment:
I personally rely on the project-wide license rather than per file.

If you prefer pre-file, we should make sure to get a CONTRIBUTING that mentions that.

Reply:
The recommendation of the FSF (via the GPLv3) is to include the license statement in all source files, and having dealt with way more FOSS licensing BS than I care to think about, I just feel better having the license in all files to avoid any confusion.

I've opened #22 and #23 to track the other changes needed so the whole project is licensed to the "predicates-rs Project Developers" and to make sure we get a CONTRIBUTING document. For this PR I think just adding the statement above is sufficient.